app/views/finance/npq/statements/show.html.erb

```diff
@@ -6,25 +6,115 @@
       <h1 class="govuk-heading-xl">National professional qualifications (NPQs)</h1>
       <span class="govuk-caption-l"><%= @statement.cpd_lead_provider.name %></span>
       <h2 class="govuk-heading-l"><%= @statement.name %></h2>
+
       <%= render Finance::Statements::NPQStatementSelector.new(current_statement: @statement) %>
-      <%= render Finance::NPQ::PaymentOverviews::PaymentOverviewSummary.new(@npq_contracts, @statement, @npq_lead_provider) %>
+
+      <div class="app-application__panel__summary">
```

Comment:
i feel this markup is bloated and could be cut down but is out of scope for this change and has been mostly left as is